QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -205,7 +205,11 @@ def convert_to_cloud_kvstyle(
         tokenizer.padding_side = "left"
 
     if tokenizer.pad_token_id is None:
-        tokenizer.pad_token_id = tokenizer.eos_token_id
+        # If Pad token is out of range of vocab size
+        if tokenizer.pad_token_id >= tokenizer.vocab_size:
+            tokenizer.pad_token_id = tokenizer.vocab_size - 1
```

Comment:
need to add the change in `load_tokenizer`?

Reply:
Then, I think there is no need of tokenizer.padding_side != "left" condition as well.